src/parser/parse.ts

```diff
@@ -5,9 +5,10 @@ import { DocNode, SyntaxKind, Token } from './types'
 const addToken = (
   stack: Token[],
   line: string,
-  isTextNode: boolean = false
+  isTextNode: boolean,
+  patternString: string
 ): void => {
-  if (line.trim() === '') {
+  if (!line) {
```

Comment:
This was actually removing spaces between multiple elements before, eg. `Hello <firstname /> <lastname />` would turn into `Hello <firstname /><lastname />`